.devcontainer/devcontainer.json

```diff
@@ -10,8 +10,9 @@
 				"twxs.cmake",
 				"ms-python.python",
 				"ms-vscode.vscode-serial-monitor",
-				"shardulm94.trailing-spaces",
-				"trond-snekvik.devicetree"
+				"trond-snekvik.devicetree",
+				"marus25.cortex-debug",
+				"marus25.cortex-debug-dp-stm32l4"
```

Comment:
What prompted the change from `cppdbg` (native Microsoft) to the "cortex-debug" extension?